salt/_pillar/metalk8s_nodes.py

```diff
@@ -92,9 +96,42 @@ def ext_pillar(minion_id, pillar, kubeconfig):
             for node in node_list.items
         )
 
+        volume_information = []
```

Comment:
Given object names are unique, it could make sense to turn this into a dict keyed by the name instead of a list, which will then ease lookup in SLSs.